BackofficeBundle/Resources/public/coffee/FullPageFormView.coffee

```diff
@@ -32,14 +32,15 @@ FullPageFormView = OrchestraView.extend(
   addEventOnForm: (event)->
     event.preventDefault()
     viewContext = @
+    viewClass = appConfigurationView.getConfiguration(@options.entityType, 'editEntity')
     $("form", @$el).ajaxSubmit
       context: button: $(".submit_form",event.target).parent()
       success: (response) ->
-        new FullPageFormView(viewContext.addOption(
+        new viewClass(viewContext.addOption(
           html: response
```

Comment:
Can you give the entityType to the view as for the other views ? (in case of override by the integrator)